### ISSUE TITLE
While writing/editing post/comment only first url in line is turned to active 

### ISSUE THREAD

**Issue by Gandalf-the-Grey** [steemit/condenser#1306]
Example:

https://steemit.com/steemit/@solarguy/im-buzzed-but-seriously#@gtg/re-solarguy-im-buzzed-but-seriously-20170403t200800536z

Moving urls to separate lines makes them "active" (i.e. clickable)

EDIT: it's for markdown mode and url written as-is
it works as expected when you explicitly provide uls as links, i.e.:
```
[link1](url://url.url) [link2](url://url.url) [link3](url://url.url)
```

**Comment by plink01001** [comment]
Has this been addressed / can this be closed?

**Comment by bnchdrff** [comment]
fixed in #1886
